golos.social/golos.social.cpp

```diff
@@ -46,6 +46,20 @@ void social::unpin(name pinner, name pinning) {
     table.modify(itr, name(), [&](auto& item){
         item.pinning = false;
     });
+
+    remove_record(pinner, pinning);
```

Comment:
`remove_record` looks strange at this point.
I think better use:
```
    if (record_is_empty(*itr))
        table.erase(itr);
...
bool record_is_empty(item) {
    return !item.pinning && !item.blocking;
}
```

in this case, we make an unnecessary modification in case of deletion (some performance degradation), but the code for determining whether the element can be deleted is assembled in one place.